runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -3263,9 +3263,9 @@ J9::Z::CodeGenerator::checkFieldAlignmentForAtomicLong()
    if (!classBlock)
       return false;
 
-   char* fieldName = "value";
+   const char* fieldName = "value";
    int32_t fieldNameLen = 5;
-   char * fieldSig = "J";
+   const char * fieldSig = "J";
    int32_t fieldSigLen = 1;
    int32_t intOrBoolOffset = self()->fe()->getObjectHeaderSizeInBytes() + self()->fej9()->getInstanceFieldOffset(classBlock, fieldName, fieldNameLen, fieldSig, fieldSigLen);
    return (intOrBoolOffset & 0x3) == 0;
```

Comment:
Given the name of the function, I suspect `0x3` ought to be `0x07` (that the offset is a multiple of 8).

Reply:
I think this part of code needs some investigation - Looking at the places where we would use this query will be while inlining method from `AtomicLong` and the value (confirmed with `sig` is long), Sincename `intOrBoolOffset` sounds  contradictory to the name of the function, and change this as a separate PR if Keith you are OK with it, I would like @matthewhall2  to verify and fix this as a separate PR ?

Reply:
@keithc-ca  ping to see what you think about above ? As I said what you pointed out is correct, but given that the this code is there since long I think a verification from @matthewhall2  is warranted on places where this query is used, so I think this can be done as a separate change (As original purpose of this PR from @matthewhall2 was to clean-up some of the warnings we saw with OpenXL compiles). 

@matthewhall2  can open up the issue and get started on it to keep track of this.

Reply:
Yes, I think it makes sense to investigate and make any necessary changes for that mask separately.